google-cloud-storage/samples/storage_create_bucket_dual_region.rb

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START storage_create_bucket_dual_region]
+require "google/cloud/storage"
+
+def storage_create_bucket_dual_region bucket_name:, region_1:, region_2:
```

Comment:
Can you start using classes to namespace the sample methods? (I know the other storage samples don't do this yet, but we have to start sometime, and now is better than later.)

Reply:
Namespaced the sample with a class.